src/main/src/test/java/org/geoserver/config/GeoServerPersistersTest.java

```diff
@@ -685,11 +715,12 @@ public void testRenameStyleWithExistingIncrementedVersion() throws Exception {
 
         File sldFile2 = new File(testData.getDataDirectoryRoot(), "styles/foostyle2.sld");
 
-        StyleInfo s = catalog.getStyleByName("foostyle");
+        StyleInfo s = catalog.getStyleByName("barstyle");
         s.setName("foostyle");
         catalog.save(s);
 
-        assertThat(sldFile, not(fileExists()));
```

Comment:
after the changes in GeoServerResourcePersister.java file, 
this line failed, so it was necessary to do the changes.  I believe just one existing test was changed